app/models/qdm/patient.rb

```diff
@@ -206,7 +204,7 @@ def id
 
     # Include '_type' in any JSON output. This is necessary for deserialization.
     def to_json(options = nil)
-      serializable_hash(methods: :_type).to_json(options)
+      serializable_hash(methods: :_type, include: :_type).to_json(options)
```

Comment:
You added a test for the `qdmPatient` of a CQM Patient, but this does not get called in that case.  It would be good to add a similar test that calls `to_json` on a QDM::Patient directly so that this line is covered.